src/crimson/osd/backfill_state.cc

```diff
@@ -260,7 +261,11 @@ BackfillState::Enqueuing::update_on_peers(const hobject_t& check)
         check == primary_bi.begin && check == peer_bi.begin) {
       if(peer_bi.objects.begin()->second != obj_v &&
           backfill_state().progress_tracker->enqueue_push(primary_bi.begin)) {
-        backfill_listener().enqueue_push(primary_bi.begin, obj_v);
+	auto &v = backfills[primary_bi.begin].first;
```

Comment:
`auto& [v, peers] = the pair`?

(Here and in other places as well).

Reply:
Ah, yes, will change it.